x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/constants.ts

```diff
@@ -50,10 +50,26 @@ export const ROLE_LABEL = i18n.translate('xpack.enterpriseSearch.roleMapping.rol
   defaultMessage: 'Role',
 });
 
+export const USERNAME_LABEL = i18n.translate('xpack.enterpriseSearch.roleMapping.usernameLabel', {
+  defaultMessage: 'Username',
+});
+
+export const EMAIL_LABEL = i18n.translate('xpack.enterpriseSearch.roleMapping.emailLabel', {
+  defaultMessage: 'Email',
+});
+
```

Comment:
Sorry, I hate to do this but this seems to be a duplicate of the new [shared/constants/labels.ts](https://github.com/elastic/kibana/pull/102826/files#diff-cea4b88d7b6ed157ae47ec4008fe2967eef144a198cde2ddc9dd919e504e5e55) file 🙈 

I think you could likely remove that file in favor of this instead.

That being said, not a blocker, we can DRY it out later!

Reply:
Sorry for the oversight. Really want to land this so adding to the tech debt sheet and will knock out ASAP